wgengine/magicsock/magicsock.go

```diff
@@ -1219,6 +1228,13 @@ func (c *Conn) derpWriteChanOfAddr(addr netaddr.IPPort, peer key.Public) chan<-
 	c.activeDerp[regionID] = ad
 	c.logActiveDerpLocked()
 	c.setPeerLastDerpLocked(peer, regionID, regionID)
+	if len(c.activeDerp) == 2 {
+		// On transition from 1 to more than 1 active DERP
+		// connections, start running the derpCleanupTicker
+		// regularly. When it drops back down to 1,
+		// closeDerpLocked will crank it back up.
+		c.derpCleanupTicker.Reset(derpInactiveCleanupTime / 3)
```

Comment:
Why divided by 3? I think I'd prefer another constant to distinguish "how often the timer runs" from "how old the connection has to be to be closed"

Reply:
done.